en/docs/install-and-setup/setup-mi-server.md

```diff
@@ -0,0 +1,41 @@
+
+Go through the following steps to setup the Micro Integrator server for VSCode. 
+
+1. Install WSO2 Micro Integrator Extension in VSCode. 
+
+2. Download the Micro Integrator 4.3.0 distribution as a ZIP file from [here](https://github.com/wso2/micro-integrator/releases/download/v4.3.0/wso2mi-4.3.0.zip).
+3. Extract the zip file and go to wso2mi-4.3.0/conf/deployment.toml
+4. Add the following configurations. 
```

Comment:
let's mention the go to deployment toml file here and add the configurations

Reply:
Done